packages/strapi-utils/lib/models.js

```diff
@@ -327,15 +327,8 @@ module.exports = {
       });
 
       if (targetName !== '*') {
-        if (association.plugin) {
-          details = _.get(
-            strapi.plugins,
-            [association.plugin, 'models', targetName, 'attributes', association.via],
-            {}
-          );
-        } else {
-          details = _.get(strapi.models, [targetName, 'attributes', association.via], {});
-        }
+        const models = strapi.db.getModelsByPluginName(association.plugin);
```

Comment:
why not use directly `strapi.db.getModel(targetName, association.plugin)` ?

Reply:
No specific reason, good catch!